src/coreclr/vm/interpreter.cpp

```diff
@@ -91,7 +91,7 @@ InterpreterMethodInfo::InterpreterMethodInfo(CEEInfo* comp, CORINFO_METHOD_INFO*
     }
 #endif
 
-#if defined(UNIX_AMD64_ABI)
+#if defined(UNIX_AMD64_ABI) || defined(HOST_LOONGARCH64)
```

Comment:
Should this be `TARGET_LOONGARCH64`? Or maybe better if custom ABI is used, create `UNIX_LOONGARCH64_ABI`.

Reply:
For LoongArch64 there is only one ABI on linux, so I used the TARGET_LOONGARCH64 as the next define,
liking the Line:100  #elif defined(HOST_ARM) || defined(HOST_AMD64)|| defined(HOST_ARM64)

Is this OK?

Reply:
I think it is ok. You can also just add `UNIX_LOONGARCH64_ABI` to the target and bring it to the same plan as other architectures: https://github.com/dotnet/runtime/blob/0206926cf3e7d45804c66cbd282de094f69ecf61/src/coreclr/clrdefinitions.cmake#L233
(it will not be wrong since there is no ambiguity)

Note, this interpreter is experimental and it just compiles, it is not used anywhere and its tests were disabled in the CI few months ago. So if interpreter.cpp is compiling on LA64 with fewer and less different tweaks, it is fine for now.

Reply:
Thanks, @kant2002 @am11 
I will do it and upload the patch later.